File: iOS/Objects/src/B4iProject/LS_frmproduct.m

#import "iCore.h"
@interface LS_frmproduct:NSObject
@end

@implementation LS_frmproduct

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[views get:@"btnreviews"].Width = (int)((100 / 100.0 * width)/2.0);
[views get:@"btnattributes"].Width = (int)((100 / 100.0 * width)/2.0);
[views get:@"btnattributes"].Left = (int)(([views get:@"btnreviews"].Width)+1.0);
[views get:@"lblattrs"].Left = (int)(([views get:@"btnattributes"].Width)+6.0);
[views get:@"pnllistnumb"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnllistnumb"].Width / 2));

}
@end

File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmproduct.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmproduct{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("btnreviews").vw.setWidth((int)((100d / 100 * width)/2d));
views.get("btnattributes").vw.setWidth((int)((100d / 100 * width)/2d));
views.get("btnattributes").vw.setLeft((int)((views.get("btnreviews").vw.getWidth())+1d));
views.get("lblreview").vw.setLeft((int)((views.get("btnreviews").vw.getWidth())-(views.get("lblreview").vw.getWidth())-6d));
views.get("pnllistnumb").vw.setLeft((int)((50d / 100 * width) - (views.get("pnllistnumb").vw.getWidth() / 2)));

}
}